app/controllers/webhooks_controller.rb

```diff
@@ -141,6 +142,24 @@ def handle_payment_intent_succeeded(
             email: payment_intent.purchaser.email
           }
         )
+      elsif project_id.present?
+        WebhookManager::DonationCreator.call(
+          {
+            seller_id: seller_id,
+            project_id: project_id,
+            payment_intent: payment_intent,
+            amount: amount
+          }
+        )
+
+        EmailManager::DonationReceiptSender.call(
```

Comment:
not setting `is_donation` and instead sending a one-off email from here--could be customized further too